### ISSUE TITLE
Fixme should trigger only on comments

### ISSUE THREAD

**Issue by scop** [PyCQA/pylint#2321]
### Steps to reproduce
1. Run pylint on a file containing this line
```python
print("# TODO this should not trigger a fixme")
```

### Current behavior

todo_issues.py:1:0: W0511: TODO this should not trigger a fixme") (fixme)

### Expected behavior

(no messages)

So pylint should be looking only into what are actual comments, not into everything that follows a `#`.

### pylint --version output

pylint 2.0.0
astroid 2.0.0.dev4
Python 3.7.0 (default, Jun 29 2018, 21:39:54) 
[GCC 7.2.0]

**Comment by PCManticore** [comment]
Thanks, this makes sense!

**Comment by PCManticore** [comment]
@sushobhit27 I'm pretty sure this still reproduces in the latest version :)

**Comment by sushobhit27** [comment]
@PCManticore ok, I labelled few other issues also after finding them working fine with 2.1.0, can you check those issues also?

**Comment by scop** [comment]
(FWIW I would find an explicit "fixed in x.y.z" label more useful than "fixed in latest version". The latter is pretty much the same as no label at all plus leaving bugs open until a release that contains the fix is out.)
